lib/icinga/clusterevents.cpp

```diff
@@ -537,12 +538,12 @@ Value ClusterEvents::AcknowledgementSetAPIHandler(const MessageOrigin::Ptr& orig
 
 	checkable->AcknowledgeProblem(params->Get("author"), params->Get("comment"),
 		static_cast<AcknowledgementType>(static_cast<int>(params->Get("acktype"))),
-		params->Get("notify"), params->Get("persistent"), params->Get("expiry"), origin);
+		params->Get("notify"), params->Get("persistent"), params->Get("change_time"), params->Get("expiry"), origin);
```

Comment:
This crashes if the instance receives a cluster message from <2.12 instances.

It needs a logical `if (params->Contains("change_time") ...`.

Reply:
* Doesn't `Dictionary#Get()` return `Empty`?
* Isn't `double(Empty)` `0.0`?